src/sentry/similarity/features.py

```diff
@@ -164,13 +164,14 @@ def classify(self, events, limit=None, thresholds=None):
                     log = (
                         logger.debug
                         if isinstance(error, self.expected_encoding_errors)
-                        else functools.partial(logger.warning, exc_info=True)
+                        else logger.warning
                     )
                     log(
                         "Could not encode features from %r for %r due to error: %r",
                         event,
                         label,
                         error,
+                        exc_info=True,
```

Comment:
i think the intent was to not log a stacktrace in case of `logger.debug`, so this is a behavioral change, but not sure it matters